### ISSUE TITLE
Escape @ only first character

### ISSUE THREAD

**Issue by francislainy** [dzieciou/curl-logger#80]
Maybe I'm missing something here, but it seems the library does not support graphQl requests?
Importing the queries printed into postman gives me some broken formatting.

```
curl 'https://my.api.com/graphql' \
  --header 'Authorization: Bearer ImyBearer_QJj-A' \
  --header 'Accept: application/json, application/javascript, text/javascript, text/json' \
  --header 'Content-Type: application/json' \
  --data-binary $'{"query":"   query {\n      getUserById(id: \"1riLgJZQHo\x40someEmail.com\", idType: \"authid\") {\n        users {\n          id\n          email\n          memberships {\n            id\n            role\n            group {\n              id,\n              name\n              account {\n                id\n              }\n              primaryGroupMemberships {\n                user {\n                  id\n                  email\n                  givenName\n                  familyName\n                  altIds {\n                    type\n                    userId\n                    value\n                  }\n                }\n                role\n              }\n            }\n          }\n        }\n      }\n    }\n"}' \
  --compressed \
  --insecure \
  --verbose 
```

<img width="930" alt="image" src="https://user-images.githubusercontent.com/19685849/179214823-de51a90f-f23e-469a-9c88-8484dd7019fd.png">

```
 public static RequestSpecification getRequestSpecification() {

        /**
         * Enables printing request as curl under the terminal as per
         * https://github.com/dzieciou/curl-logger
         */
        Options options = Options.builder().printMultiliner().updateCurl(curl -> curl.removeHeader("Host").removeHeader("User-Agent").removeHeader("Connection")).build();

        RestAssuredConfig config =
                CurlRestAssuredConfigFactory.createConfig(options).objectMapperConfig(new ObjectMapperConfig().jackson2ObjectMapperFactory(new Jackson2ObjectMapperFactory() {
                    @Override
                    public ObjectMapper create(Type type, String charset) {
                        ObjectMapper om = new ObjectMapper().findAndRegisterModules();
                        om.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
                        om.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
                        return om;
                    }

                }));

        return given().config(config).contentType(ContentType.JSON).accept(ContentType.JSON).urlEncodingEnabled(false).when().log().everything();
    }
```

```
<rest-assured.version>4.5.1</rest-assured.version>
<json-schema-validator.version>5.1.1</json-schema-validator.version>
<http.client.version>5.1.2</http.client.version>
<curl.logger.version>2.1.1</curl.logger.version>
```

Thank you.

**Comment by francislainy** [comment]
I think we can close this issue. I think the issue is more to do with Postman. Tested it against Insomnia and it worked (after I replaced \x40 character with @ for the email. Any way to configure the log not to need the manual conversation and keep the proper @ character, since this is what we're sending to the request?)

**Comment by dzieciou** [comment]
I will look into that issue.

**Comment by dzieciou** [comment]
Fixed by #82.
